packages/engine/errors/7061.md

```diff
@@ -0,0 +1,34 @@
+---
+original: 'A mapped type may not declare properties or methods.'
+excerpt: 'You can only create a mapped type using a type alias.'
```

Comment:
```suggestion
excerpt: 'You can only create a mapped type using `type`.'
```

Reply:
Hi Matt,


Sure here is a link to a playground where this happens. https://www.typescriptlang.org/play?#code/JYOwLgpgTgZghgYwgAgApwM4LgGwHJwC2KA3gFDLIBiwUGYBxAXMvVKAOYDcFyAMpgZEILNpx4BfMmTABPAA4oAwsJwqMERhADSEWRmQBeZAFUQ2ecDC5gALwgAeANZ6A9jDSZs+YQD4eZKCQsIjKquqawsjklADaurLIoMgqxGqYkcQJGAC6omDsINxkUkA